openmp/libomptarget/plugins-nextgen/common/src/PluginInterface.cpp

```diff
@@ -362,7 +362,10 @@ struct RecordReplayTy {
   }
 };
 
-static RecordReplayTy RecordReplay;
+RecordReplayTy &getRecordReplay() {
```

Comment:
I think I just need to move this into the plugin initialization code. @jdoerfert could we just move this into the plugin instead? I'm assuming we don't need this to actually be global and it would make the lifetime much nicer.